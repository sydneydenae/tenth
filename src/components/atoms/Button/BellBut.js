import React from 'react'
import Bell from '../Icons/Bell'

export default function(){
    return(
        <button className="bg-gray-800 p-1 rounded-full text-gray-400 hover:text-white focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-offset-gray-800 focus:ring-white">
            <span className="sr-only">View notifications</span>
            <Bell/>
        </button>

    )
}