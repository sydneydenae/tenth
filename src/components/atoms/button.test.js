import React from "react"
import Button from "./button"
import renderer from "react-test-renderer"

describe("Button", () => {
    it("renders correctly", () => {
      const tree = renderer
        .create(<Button />)
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
  })