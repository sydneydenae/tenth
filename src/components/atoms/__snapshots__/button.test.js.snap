// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<button
  className="inline-flex items-center px-4 py-2 border border-transparent text-sm font-medium rounded-md shadow-sm text-white bg-mmb-500 hover:bg-mmb-600 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-mmb-500"
  type="button"
>
  Click Me!
</button>
`;
